Source/CBL_Replicator.h

```diff
@@ -77,8 +77,10 @@ extern NSString* CBL_ReplicatorStoppedNotification;
 /** Called by CBLDatabase to notify active replicators that it's about to close. */
 - (void) databaseClosing;
 
-#if DEBUG // for unit tests
+/** Current lastSequence tracked by the replicator. */
 @property (readonly) id lastSequence;
```

Comment:
In the implementations of CBL_Replicator, `lastSequence` is only synthesized in a debug build, so I think a release build is going to fail with an error about a missing property implementation. You'll need to move the `@synthesize` out of `#if DEBUG`.